docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -14,17 +14,22 @@ Linux operating system
 
 * 64-bit required
 * newer release preferred, especially when running on containers
-* adequate ulimits for the user that runs the Trino process. These limits
-  may depend on the specific Linux distribution you are using. The number
-  of open file descriptors needed for a particular Trino instance scales
-  as roughly the number of machines in the cluster, times some factor
-  depending on the workload. We recommend the following limits, which can
-  typically be set in ``/etc/security/limits.conf``:
+* adequate ulimits for the user that runs the Trino process. These limits may
+  depend on the specific Linux distribution you are using. The number of open
+  file descriptors needed for a particular Trino instance scales as roughly the
+  number of machines in the cluster, times some factor depending on the
+  workload. The ``nofile`` limit sets the maximum number of file descriptors
+  that a process can have, while the ``nproc`` limit restricts the number of
+  processes a user can create. We recommend setting limits to the following
+  values at a minimum. Typically, this configuration is located in
+  ``/etc/security/limits.conf``:
 
   .. code-block:: text
 
       trino soft nofile 131072
       trino hard nofile 131072
+      trino soft nproc 128000
```

Comment:
LGTM!

>  I just dont know about the actual value being proposed...

128k should be a safe value, unless the machine is very, very large. I've seen issues with `nproc = 4096` on 8 core machines. Scaling that up for a 96 core machine: `4096 x 96/8 = 49152 `, which is still well below 128k.

 `nproc` is sometimes set quite low ootb in enterprise settings (e.g. 1024 or 4096) so I do agree that this is a good addition to the docs.

The error message reported when `nproc` is too low should be 
```
java.lang.OutOfMemoryError : unable to create new native Thread
```
in case you want to include that in the documentation.